src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.config;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataInitializer {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    public DataInitializer(RoleService roleService, UserService userService) {
+        this.roleService = roleService;
+        this.userService = userService;
+    }
+
+    @PostConstruct
+    public String injectRoles() {
```

Comment:
Do we need return type in init() methods?

Reply:
It is much easier to understand if injection is successful or not if you have a message in browser.

Reply:
Ok I understood. This message was there because previously I didn't use a post construct annotation. Now it is redundant.